spec/datadog/ci/test_spec.rb

```diff
@@ -157,6 +157,70 @@
           .to eq(test_type)
       end
     end
+
+    context 'with environment runtime information' do
+      context 'for the architecture platform' do
+        subject(:tag) do
+          set_tags!
+          span.get_tag(Datadog::CI::Ext::Test::TAG_OS_ARCHITECTURE)
+        end
+
+        it { is_expected.to eq('x86_64') }
+      end
+
+      context 'for the OS platform' do
+        subject(:tag) do
+          set_tags!
+          span.get_tag(Datadog::CI::Ext::Test::TAG_OS_PLATFORM)
+        end
+
+        context 'with Linux', if: PlatformHelpers.linux? do
+          it { is_expected.to start_with('linux') }
+        end
+
+        context 'with Mac OS', if: PlatformHelpers.mac? do
+          it { is_expected.to start_with('darwin') }
+        end
```

Comment:
The specs seems to "respect" the ability of each language to return reliable output.
Because it's very hard for me to know what the output of `RbConfig::CONFIG['host_os']` will be on every installation, I'm opting to keep it unprocessed, and refined as we see fit.

Reply:
Fair enough. We can also do this in the backend if we want, so no problem :)

(references to "darwin" nowadays remind me of how so many browsers still claim they are "khtml" in the user agent, and nobody still knows what that is). Ahh legacy.